plaidml/testenv.h

```diff
@@ -36,7 +36,8 @@ class TestFixture : public ::testing::Test {
     EXPECT_EQ(a, b);
   }
 
-  void compareElements(float a, float b) { EXPECT_NEAR(a, b, (fabs(a) + fabs(b)) / 10000.0); }
+  // Higher tolerance for full network computations
+  void compareElements(float a, float b) { EXPECT_NEAR(a, b, (fabs(a) + fabs(b)) / 100.0); }
```

Comment:
I'm not sure this is a good idea, @tzerrell ?

Reply:
To provide some insight here: `compareElements` is generally failing for values `a` and `b` that are sufficiently small - around `1e-05` and smaller. In those cases, I think that this test may be running into precision issues.

Reply:
I think a higher tolerance for full network comparisons is reasonable, but this is IMO too high a tolerance and also it applies to anything using compareElements on floats, not just full networks. My suggestions:
* Rather than changing the global tolerance, add an optional tolerance parameter to this function. Then on the specific tests which need looser tolerance, provide what's needed for each specific test. Perhaps both an absolute and relative tolerance parameter (see below)
* It may be necessary to add an absolute tolerance component to the test; it's not uncommon to see outputs like 3.25e-33 vs. 8.12e-29 for different implementations computing "approximately zero", and that's fine for ML workloads that are OK with reordering floating point reductions (i.e. basically all of them), but the relative error is huge. A possible formula with both absolute and relative tolerance terms:
```
rtol * ((fabs(a) + fabs(b)) / 2.) + atol
```
This is about as close as you can get to the all_close formula we from numpy while being symmetric in `a` and `b`. See those docs for more: https://numpy.org/doc/stable/reference/generated/numpy.allclose.html